docs/data/base/components/select/UnstyledSelectObjectValuesForm.tsx

```diff
@@ -126,11 +127,12 @@ const StyledPopper = styled(PopperUnstyled)`
   z-index: 1;
 `;
 
-const Header = styled('h1')(
+const Label = styled('label')(
```

Comment:
Remove the `h1`. I think that the only demo that can justify having an h1 is `<Typography variant="h1">`.

Reply:
I have also used this opportunity to fix the a11y:

- **VoiceOver** doesn't read this as a label. https://mui.com/base/react-select/#object-values:

<img width="966" alt="Screenshot 2022-08-28 at 13 38 46" src="https://user-images.githubusercontent.com/3165635/187072322-1e2b1187-3ee4-4644-9c99-004649a96bd2.png">

But on https://mui.com/material-ui/react-select/#basic-select, it does:

<img width="836" alt="Screenshot 2022-08-28 at 13 38 33" src="https://user-images.githubusercontent.com/3165635/187072308-680c7443-90ca-4550-ab83-73e6d1efabfe.png">

Reply:
Then I think that it depends on the perspective, the specs say

> The label element represents a caption in a user interface

https://html.spec.whatwg.org/multipage/forms.html#the-label-element

So are we writing the caption for the developer or the end user? In https://mui.com/material-ui/react-autocomplete/ we assume that it's the developer.

In any case, I think that we miss a section in the docs about how to add a label, this should probably be high in the TOCs as I assume needed for most use cases.